etlman/projects/views.py

```diff
@@ -76,3 +75,135 @@ def new_project_view(request):
     form = ProjectForm()
     context = {"form": form, "username": username}
     return render(request, "projects/new_project.html", context)
+
+
+@authorize(user_is_project_collaborator)
+def pipeline_list(request, project_id):
+    project = get_object_or_404(Project, pk=project_id)
+    pipelines = Pipeline.objects.filter(project=project)
+    context = {"pipeline_list": pipelines, "current_project": project}
+    return render(request, "projects/pipeline_list.html", context)
+
+
+@authorize(user_is_project_collaborator)
+def new_pipeline_step1(request, project_id):
+    project = get_object_or_404(Project, pk=project_id)
+    username = request.user.username
+    # Form functionality
+    if request.method == "POST":
+        form_pipeline = PipelineForm(request.POST)
+        form_datainterface = DataInterfaceForm(request.POST)
+        if form_pipeline.is_valid() and form_datainterface.is_valid():
+
+            filled_datainterface = form_datainterface.save(commit=False)
+            filled_datainterface.name = request.POST.getlist("name")[-1]
```

Comment:
Please use the form `prefix`

Reply:
This was solved after the review, please check updated version